sample-xml/app/src/main/java/co/nimblehq/sample/xml/ui/screens/second/SecondFragment.kt

```diff
@@ -0,0 +1,36 @@
+package co.nimblehq.sample.xml.ui.screens.second
+
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import co.nimblehq.sample.xml.databinding.FragmentHomeBinding
+import co.nimblehq.sample.xml.extension.provideViewModels
+import co.nimblehq.sample.xml.model.UiModel
+import co.nimblehq.sample.xml.ui.base.BaseFragment
+import co.nimblehq.sample.xml.ui.screens.MainNavigator
+import dagger.hilt.android.AndroidEntryPoint
+import timber.log.Timber
+import javax.inject.Inject
+
+@AndroidEntryPoint
+class SecondFragment : BaseFragment<FragmentHomeBinding>() {
```

Comment:
Should we create a new layout file for `SecondFragment` in this PR instead of using the one from `HomeFragment` 🙏🏼 We can just create a blank layout for now IMO

Reply:
@chornerman Let's do it in Part2. For now I'll add TODO comment.